src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -2851,6 +3322,40 @@ public static unsafe Vector256<T> ToVector256Unsafe<T>(this Vector128<T> vector)
             return result;
         }
 
+        [Intrinsic]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static Vector128<T> Truncate<T>(Vector128<T> vector)
+        {
+            if ((typeof(T) == typeof(byte))
+             || (typeof(T) == typeof(short))
+             || (typeof(T) == typeof(int))
+             || (typeof(T) == typeof(long))
+             || (typeof(T) == typeof(nint))
+             || (typeof(T) == typeof(nuint))
+             || (typeof(T) == typeof(sbyte))
+             || (typeof(T) == typeof(ushort))
+             || (typeof(T) == typeof(uint))
+             || (typeof(T) == typeof(ulong)))
+            {
+                return vector;
+            }
+            else
```

Comment:
We don't want or need to do something here like what TensorPrimitives's TruncateOperator currently does? Same for Round above?

Reply:
No, because this is just the fallback path used when the underlying runtime doesn't accelerate `Vector128` itself.

Historically we've had such fallbacks explicitly handle the `no-ops` (which helps out Mono, for example) and otherwise defer down to the next smaller vector size (which often is accelerated; but otherwise does a scalar loop that can be unrolled).